tests/ballerina-integration-test/src/test/resources/http/httpservices/13_http_header_test.bal

```diff
@@ -22,40 +22,33 @@ service<http:Service> headerService bind headerServiceEP {
         req.addHeader("core", "bbb");
 
         var result = stockqEP -> get("/sample/stocks", message = untaint req);
-        match result {
-            http:Response clientResponse => {
-                _ = conn -> respond(clientResponse);
-            }
-            error err => {
-                _ = conn -> respond(err.reason());
-            }
+        if (result is http:Response) {
+            _ = conn->respond(result);
+        } else if (result is error) {
+            _ = conn->respond(result.reason());
         }
     }
 
     id (endpoint conn, http:Request req) {
         http:Response clntResponse = new;
-        var reply = stockqEP -> forward("/sample/customers", req);
-
-        match reply {
-            http:Response clientResponse => {
-                json payload = {};
-                if (clientResponse.hasHeader("person")) {
-                    string[] headers = clientResponse.getHeaders("person");
-                    if (headers.length() == 2) {
-                        payload = {header1:headers[0], header2:headers[1]};
-                    } else {
-                        payload = {"response":"expected number of 'person' headers not found"};
-                    }
+        var clientResponse = stockqEP -> forward("/sample/customers", req);
+        if (clientResponse is http:Response) {
+            json payload = {};
+            if (clientResponse.hasHeader("person")) {
+                string[] headers = clientResponse.getHeaders("person");
+                if (headers.length() == 2) {
+                    payload = {header1:headers[0], header2:headers[1]};
                 } else {
-                    payload = {"response":"person header not available"};
+                    payload = {"response":"expected number of 'person' headers not found"};
                 }
-                http:Response res = new;
-                res.setJsonPayload(untaint payload);
-                _ = conn -> respond(res);
-            }
-            error err => {
-                _ = conn -> respond(err.reason());
+            } else {
+                payload = {"response":"person header not available"};
             }
+            http:Response res = new;
```

Comment:
These two lines are redundant, we can simply pass the payload to the respond()??